src/js/components/ToggleGroup/ToggleGroup.js

```diff
@@ -1,10 +1,15 @@
 import React, { useCallback, useState, useRef, useMemo } from 'react';
+import styled from 'styled-components';
 import { Box } from '../Box';
 import { Keyboard } from '../Keyboard';
 import { ToggleGroupPropTypes } from './propTypes';
 import { StyledButton } from './StyledToggleGroup';
 import { useThemeValue } from '../../utils/useThemeValue';
 
+const StyledToggleGroupBox = styled(Box)`
```

Comment:
Should this go in the StyledToggleGroup file jsut to keep the styled components all contained in one place?

Reply:
I can move it yeah just saw only Button was in the styled file so I put this here but I can move it

Reply:
moved